lib/client/api.go

```diff
@@ -703,7 +703,9 @@ func (tc *TeleportClient) Login() error {
 
 	// generate a new keypair. the public key will be signed via proxy if our password+HOTP
 	// are legit
-	priv, pub, err := native.New().GenerateKeyPair("")
+	keygen := native.New()
+	defer keygen.Close()
+	priv, pub, err := keygen.GenerateKeyPair("")
```

Comment:
we need to make sure that Pool is only enabled when web proxy uses it, otherwise it should be turned off. or we may simply make it a property of auth sever